src/Gui/SoFCColorBar.cpp

```diff
@@ -78,6 +80,32 @@ void SoFCColorBarBase::GLRenderBelowPath(SoGLRenderAction *  action)
     SoSeparator::GLRenderBelowPath(action);
 }
 
+float SoFCColorBarBase::getBoundingWidth(const SbVec2s& size)
+{
+    // These are the same camera settings for front nodes as defined in the 3d view
+    SoOrthographicCamera* cam = new SoOrthographicCamera;
+    cam->position = SbVec3f(0, 0, 5);
+    cam->height = 10;
+    cam->nearDistance = 0;
+    cam->farDistance = 10;
```

Comment:
same here

Reply:
The near and far distances are not that important either because the geometry nodes that are inside the foreground nodes of the viewer usually use z=0.